content/vendor/cli/tokens.md

```diff
@@ -19,12 +19,16 @@ Tokens are primarily used with vendor CLI commands for automated customer, chann
 
 ## Team Tokens
 
+Team tokens should be used for [Vendor CLI](https://help.replicated.com/api/replicated-vendor-cli/) authorization.
```

Comment:
@jeffreygolden How would you feel about moving from 'should' to 'are'? It's less words and technically accurate (though much more blunt). Somewhat trying to keep [these](https://github.com/replicatedhq/kots.io/blob/master/CONTRIBUTING.md) in mind but maybe I am taking it too far.

Reply:
These changes are in my last commit; going to resolve this comment.